library/base/alchitry-au/Lucid/Reg Interface/source/au_top.luc

```diff
@@ -0,0 +1,60 @@
+module au_top (
+    input clk,              // 100MHz clock
+    input rst_n,            // reset button (active low)
+    output led [8],         // 8 user controllable LEDs    
+    input usb_rx,           // USB->Serial input
+    output usb_tx           // USB->Serial output
+  ) {
+  
+  sig rst;                  // reset signal
+  
+  .clk(clk) {
+    // The reset conditioner is used to synchronize the reset signal to the FPGA
+    // clock. This ensures the entire FPGA comes out of reset at the same time.
+    reset_conditioner reset_cond;
+    
+    .rst(rst) {
+      #BAUD(1000000), #CLK_FREQ(100000000) {
+        uart_rx rx;
+        uart_tx tx;
+      }
+      
+      reg_interface reg;

```

Comment:
Clock frequency at different speed for Au vs default set for registry interface. Should be:

`reg_interface reg(#CLK_FREQ(100000000));`